test/enclave_wrangler_tests/test_utils2.py

```diff
@@ -0,0 +1,47 @@
+"""Tests
+
+How to run:
+    python -m unittest discover
+"""
+import os
+import sys
+import unittest
+from pathlib import Path
+
+TEST_DIR = os.path.dirname(__file__)
+PROJECT_ROOT = Path(TEST_DIR).parent.parent
+sys.path.insert(0, str(PROJECT_ROOT))
+from backend.db.utils import get_db_connection, insert_fetch_status, run_sql, sql_query
+
+TEST_INPUT_DIR = os.path.join(TEST_DIR, 'input', 'test_enclave_wrangler')
+TEST_SCHEMA = 'test_n3c'
+
+
+# todo: temporarily naming this 'utils2' to make merge conflicts easier until merge Hope's new PR, then will merge this
+#  into that file.
+class TestEnclaveWranglerUtils2(unittest.TestCase):
+
+    def test_insert_fetch_status(self):
```

Comment:
Not sure if the most useful unit test in the world, but I did verify in an easy way that things were working correctly, so probably good to have this.